packages/zwave-js/src/lib/controller/Inclusion.ts

```diff
@@ -132,6 +133,11 @@ export type InclusionOptions =
 		 * This is not recommended due to the overhead caused by S0.
 		 */
 		forceSecurity?: boolean;
+
+		/**
+		 * Force long range. If not provided, will default to long range iff the controller supports it, and not otherwise.
+		 */
+		isLongRange?: boolean;
```

Comment:
This goes back to "how will the UI specify it want's to do long range inclusion or not". Whatever options there are should probably go here, but I'd like to hear from the maintainers how they'd like it to work.